providers/aws/services/iam/iam_service.py

```diff
@@ -237,6 +242,83 @@ def __list_mfa_devices__(self):
         except Exception as error:
             logger.error(f"{self.region} -- {error.__class__.__name__}: {error}")
 
+    def __list_attached_user_policies__(self):
+        try:
+            for user in self.users:
+                attached_user_policies = []
+                get_user_attached_policies_paginator = self.client.get_paginator(
+                    "list_attached_user_policies"
+                )
+                for page in get_user_attached_policies_paginator.paginate(
+                    UserName=user.name
+                ):
+                    for policy in page["AttachedPolicies"]:
+                        attached_user_policies.append(policy)
+
+                user.attached_policies = attached_user_policies
+
+        except Exception as error:
+            logger.error(f"{self.region} -- {error.__class__.__name__}: {error}")
+
+    def __list_inline_user_policies__(self):
+        try:
+            for user in self.users:
+                inline_user_policies = []
+                get_user_inline_policies_paginator = self.client.get_paginator(
+                    "list_user_policies"
+                )
+                for page in get_user_inline_policies_paginator.paginate(
+                    UserName=user.name
+                ):
+                    for policy in page["PolicyNames"]:
+                        inline_user_policies.append(policy)
+
+                user.inline_policies = inline_user_policies
+
+        except Exception as error:
+            logger.error(f"{self.region} -- {error.__class__.__name__}: {error}")
+
+    def __get_support_roles__(self):
```

Comment:
I think it could be better to rename this function to something like `__get_entities_attached_to_support_roles`, what do you think?

Reply:
Done